sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -149,8 +149,10 @@ public abstract partial class Operation<T> : Azure.Operation where T : notnull
         protected Operation() { }
         public abstract bool HasValue { get; }
         public abstract T Value { get; }
+        public virtual Azure.Response<T> WaitForCompletion(Azure.Core.Delay delayStrategy, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
Any reason this overload wasn't added to `Operation.WaitForCompletionResponse`?

Reply:
Will add.